### ISSUE TITLE
`events.emit` does not send payload

### ISSUE THREAD

**Issue by remorses** [voxelize/voxelize#30]
For example this will print stuff without the payload, even on the server the payload is missing
```ts

const events = new VOXELIZE.Events()

network.register(events)

events.on('stuff', (payload) => {
    console.log('stuff', payload)
})
events.emit('stuff', 'hello')
```

**Comment by shaoruu** [comment]
[Fixed in 0.1.192 ✅](https://github.com/voxelize/voxelize/commit/788b991e251b61f5cea1d8cd110a64b14dbedd43). The bug was that i temporarily started using google's Struct api for a while back in early 2023, but forgot to migrate back events back to using strings. Let me know if it's still not working!

**Comment by remorses** [comment]
Awesome thank you!
